gcc-4.8/post-link.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+if [ "$(uname)" != "Darwin" ]; then
```

Comment:
Why is it needed? As far as I can tell, all it does is test the compiler. I haven't heard of any issues with the existing gcc package on OS X.